src/scripts/mutual_checker.js

```diff
@@ -12,15 +12,32 @@ const hiddenAttribute = 'data-mutual-checker-hidden';
 const mutualsClass = 'from-mutual';
 const postAttributionSelector = `header ${keyToCss('attribution')} > span:not(${keyToCss('reblogAttribution')}) a`;
 
+const styleElement = buildStyle();
+const buildCss = () => `${keyToCss('notification')}:not([data-mutuals]) { display: none !important; }`;
```

Comment:
You can set the contents of the style element directly if they don't need to change based on preferences:

```suggestion
const styleElement = buildStyle(`${keyToCss('notification')}:not([data-mutuals]) { display: none !important; }`);
```